src/components/cc-addon-admin/cc-addon-admin.js

```diff
@@ -108,10 +108,11 @@ export class CcAddonAdmin extends LitElement {
         ${!loadingError ? html`
 
           <cc-block-section>
-            <div slot="title">${i18n('cc-addon-admin.addon-name')}</div>
+            <div slot="title">${i18n('cc-addon-admin.heading.name')}</div>
             <div slot="info"></div>
             <div class="one-line-form">
               <cc-input-text
+                label="${i18n('cc-addon-admin.input.name')}"
```

Comment:
all translations of `cc-addon-admin` are visible in the story (no hidden label) so you can skip this and only check visually.